src/core/Akka/Util/MatchHandler/ILambdaExpressionCompiler.cs

```diff
@@ -22,15 +22,6 @@ internal interface ILambdaExpressionCompiler
         /// <param name="expression">The expression to compile</param>
         /// <returns>A delegate containing the compiled version of the lambda.</returns>
         Delegate Compile(LambdaExpression expression);
-
-#if !CORECLR
-        /// <summary>
-        /// Compiles the lambda into a method definition.
-        /// </summary>
-        /// <param name="expression">The expression to compile</param>
-        /// <param name="method">A <see cref="MethodBuilder"/> which will be used to hold the lambda's IL.</param>
-        void CompileToMethod(LambdaExpression expression, MethodBuilder method);
```

Comment:
Byterot, these codes were never compiled before the cleanup (since we moved to single `netstandard2.0` platform target)